migrations/versions/96a8d065e5ea_.py

```diff
@@ -69,6 +69,16 @@ def upgrade():
     sa.ForeignKeyConstraint(['contributor_id'], ['contributor.id'], ),
     sa.PrimaryKeyConstraint('id')
     )
+    op.create_table('api_keys',
```

Comment:
Idea being there are many API-keys give out. When requests come in, look to see if there is a matching api_key entry in our db